package.json

```diff
@@ -13,6 +13,7 @@
     "express-socket.io-session": "^1.3.5",
     "express-ws": "^4.0.0",
     "js-cookie": "^2.2.0",
+    "lint": "^0.7.0",
```

Comment:
Was this package accidentally installed?

Reply:
Probably, I'll create an issue an remove it now